README.md

```diff
@@ -60,7 +60,11 @@ $ scoop install https://raw.githubusercontent.com/appwrite/sdk-for-cli/master/sc
 Once the installation completes, you can verify your install using
 ```
 $ appwrite -v
+<<<<<<< Updated upstream
 5.0.1
+=======
+5.0.2
+>>>>>>> Stashed changes
```

Comment:
the merge conflict wasnt resolved properly

Reply:
@christyjacob4, fixed! Thanks for catching that!